### PR TITLE
Add 5-21-2020 meeting transcript

### PR DESCRIPTION
**Pull request by kaicataldo**
Rather than dealing with setting up a .NET environment and using https://github.com/Tyrrrz/DiscordChatExporter, I threw together a quick little script to generate this (you can see the code [here](https://github.com/kaicataldo/discord-transcript-generator)) that we can use going forward. It's written in Node, so should be easy enough for anyone on the TSC to run it. If we want to add this as an `npm` script in this repo, I can definitely do that (either publishing it or just inlining it), or folks can just clone the repo and run it locally as well.

It will require that the person running the script has access to the shared secrets for Discord API, so we would have to figure out how we want to do that. It would be great if whoever is taking notes can generate the transcript as part of the meeting notes PR.

Comments on the format? Happy to update the script if need be.

### CODE REVIEW

File: transcripts/5-21-2020.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# 5/21/2020 ESLint TSC Meeting Transcript
+
+**kaicataldo:** Hello 👋
+**nzakas:** Hi there! Welcome to our Discord channel for TSC meetings!
+**btmills:** 👋 smells like fresh paint in here!
+**nzakas:** Hehe
+**nzakas:** Looks like <@!235597315367239700> can't join us, so we're ready to get started.
```
Comment (kaicataldo) [inline transcripts/5-21-2020.md:7]: Might be good to replace these user id's with their Discord username. I'll take a look at that tomorrow.

**Comment by nzakas** [comment]
Looks like a good start. One issue is that single line breaks in markdown are swallowed, the rendered View doesn’t have any line breaks and so it’s hard to read. You may need to insert line breaks manually to adjust for reading.

**Comment by kaicataldo** [comment]
I tried adding both 2 spaces and `<br />` at the end of each line. This did fix the issue of the rendered markdown not adding newlines, but it still didn't fix the issue of incorrect formatting after a list (example [here](https://github.com/eslint/tsc-meetings/blob/5730b8cfdf5129ce807a924742a8aaf3fcd7cb2e/transcripts/5-21-2020.md)). Adding a padded newlines ensures that's it all renders correctly, and I think it makes it easier to read to.

**Comment by kaicataldo** [comment]
Updated! Was able to fine-tune the script a bit more as well. I'll probably end up publishing it so that it can be installed globally/used programmatically in the future when we want to automate it.

**Comment by kaicataldo** [comment]
@nzakas Do you have any other concerns?

**Pull request merged**